strawberryfields/api/result.py

```diff
@@ -107,8 +107,9 @@ def state(self):
             raise AttributeError("The state is undefined for a stateless computation.")
         return self._state
 
-    def __str__(self):
+    def __repr__(self):
         """String representation."""
-        return "Result: {} subsystems, state: {}\n samples: {}".format(
-            len(self.samples), self.state, self.samples
+        shots, modes = self.samples.shape
+        return "<Result: num_modes={}, shots={}, contains state={}>".format(
```

Comment:
So when working in a Jupyter Notebook, one could first have `print(Result)` which would then include the information about e.g. `Result.is_stateful==True`. And then this could be followed simply just by returning `Result.state`, right?

Reply:
Yep!